README.md

```diff
@@ -1,183 +1,196 @@
-# React Native Base - TypeScript
-
-![GHA CI Test](https://github.com/rootstrap/react-native-base/workflows/CI%20Test/badge.svg)
-[![Maintainability](https://api.codeclimate.com/v1/badges/8bb29bcea21bb5dda316/maintainability)](https://codeclimate.com/github/rootstrap/react-native-base/maintainability)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/8bb29bcea21bb5dda316/test_coverage)](https://codeclimate.com/github/rootstrap/react-native-base/test_coverage)
```

Comment:
While we don't configure Sonarqube I feel we should keep the codeclimate badges

Reply:
Agreed! We are keeping them. You can find them in line 8.